packages/api/src/utils/file-utils.ts

```diff
@@ -0,0 +1,93 @@
+const DEFAULT_IMAGE_WIDTH = 2560;
+const DEFAULT_IMAGE_QUALITY = 0.9;
+
+type TCompressImageOption = {
+    maxWidth?: number;
```

Comment:
React.CSSProperties['maxWidth']; ?

Reply:
this generic property returns a string or number for css property so '2rem' '20%'. The resizing algorithm only supports number